crates/testing/src/sdk.rs

```diff
@@ -7,11 +7,16 @@ use std::{collections::HashSet, fs::create_dir_all, sync::Mutex};
 use crate::invoke_cli;
 use crate::modules::{module_path, CompiledModule};
 use std::path::Path;
+use tempfile::TempDir;
 
 pub fn ensure_standalone_process() {
     lazy_static! {
-        static ref JOIN_HANDLE: Mutex<Option<JoinHandle<()>>> =
-            Mutex::new(Some(std::thread::spawn(|| invoke_cli(&["start"]))));
+        // TODO: this leaks the tempdir, as dtors for lazy_statics never run.
+        static ref STANDALONE_PATH: TempDir = TempDir::with_prefix("stdb-sdk-test").expect("Failed to create tempdir");
```

Comment:
I'd inline this into initializer below - it doesn't need to be its own lazy static I think?

Reply:
Done.

Reply:
Alternatively, wrap it into `ManuallyDrop` to make the intentional leak clearer.

Reply:
Yes, as written this will drop the TempDir at the end of the block.

Reply:
I don't think that's true; I believe `into_path` leaks the tempdir.

Reply:
Ah gotcha, I think this is new in the tempdir to tempfile update. too sensitive to this after too many footgun bugs around the tempdir crate, sorry. lgtm